inspector/src/components/actionTabs/tabs/propertyGrids/animations/curveEditorComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+
+import { DraggableWindowComponent } from './draggableWindowComponent';
+
+interface ICurveEditorComponentProps {
+    isOpen: boolean;
+    close: Function;
+}
+
+export class CurveEditorComponent extends React.Component<ICurveEditorComponentProps, {isOpen: boolean}> {
+
+    constructor(props: ICurveEditorComponentProps) {
+        super(props);
+    }
+
+    render() {

```

Comment:
Does this window can be moved to another screen? If not we need to do what I did for NME (use a regular window)

Reply:
Let me check how the NME works. 👍

Reply:
For this one, you mean a de-attachable window? Like 
![image](https://user-images.githubusercontent.com/704682/79022810-69b9d380-7b44-11ea-96bb-754572ba339b.png)

or 

![image](https://user-images.githubusercontent.com/704682/79022834-7807ef80-7b44-11ea-9d18-86b2c834c6b0.png)

Reply:
Yep